metrics/network/remote_network/remote-networking-test-common.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This file contains functions that are shared among the networking
```

Comment:
it seems that these files will be in another directory `(remote_network)`, I am not sure why, these seems to be common functions, maybe these should be in `/lib/network-comon.bash`